renderer/views/torrent-list.js

```diff
@@ -153,7 +153,7 @@ function TorrentList (state) {
         <i.button-round.icon.play
```

Comment:
@feross renamed this file exactly opposite in https://github.com/feross/webtorrent-desktop/commit/6e240b3fd492481eb6c7fc464468ceb15edca380?